sharktank_models/test_suite/regression_tests/README.md

```diff
@@ -0,0 +1,52 @@
+## Regression tests
+
+### Adding your own model
+
+- To add your own model, create a directory under `regression_tests` and add JSON files that correspond to the submodels. Please follow the [JSON file schema in this README file](#required-and-optional-fields-for-the-json-model-file)
+
+### How to run
+
+- Example command to run all submodels for a specific model
+
+```
+python sharktank_models/test_suite/regression_tests/run_all_submodels.py --model sdxl
+```
+
+- Example command to run a specific model and submodel
+
+```
+pytest sharktank_models/test_suite/regression_tests/test_model_threshold.py \
+    -rpfE \
+    --log-cli-level=info \
+    --capture=no \
+    --timeout=600 \
+    --durations=0 \
+    --model-name sd3 \
+    --submodel-name vae
+```
+
+### Required and optional fields for the JSON model file
```

Comment:
Docs here look duplicated with the benchmarks? Should there be one doc shared between each, instead of duplicating?

Reply:
Okay, keeping two similar READMEs sounds good to me in that case.